roles/openshift_health_checker/test/search_journalctl_test.py

```diff
@@ -0,0 +1,199 @@
+import pytest
+import search_journalctl
+import sys
+
+from time import time
+
+
+SAVED_TIMESTAMPS = list()
+
+if sys.version_info > (3, 0):
+    RE_ERROR_UNEXPECTED_END_REGEXP = "unterminated character set"
+else:
+    RE_ERROR_UNEXPECTED_END_REGEXP = "unexpected end of regular expression"
+
+
+def canned_search_journalctl(get_log_input=None):
+    """Create a Curator check object with canned get_log_input method"""
+    module = search_journalctl  # fails if a module is actually invoked
+    if get_log_input:
+        module._get_log_input = get_log_input
+    return module
+
+
+def get_timestamp_ms(modifier=0, saveto=None):
```

Comment:
I could address the `saveto` parameter by just creating a global "log entry" at the start of the test that I re-use as part of "expected" sets. Did you also have concerns about the modifier? I use that to [create fake older log entries](https://github.com/openshift/openshift-ansible/pull/4316/files#diff-a515f88fdca679286abb67d0bc994c98R169)